File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/ResumeStagedRolloutTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.PublishingService;

public class ResumeStagedRolloutTask extends BaseTask {
	
	public ResumeStagedRolloutTask() {
		setDescription("Resume the current staged rollout");
	}
	
	@Override
	protected void onExecute(App app) {
		new PublishingService().resumeStagedRollout(app);
	}
	
}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/ResumeStagedRolloutTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

open class ResumeStagedRolloutTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().resumeStagedRollout(app)
    }

    init {
        description = "Resume the current staged rollout"
    }
}
